about_processes.exs

```diff
@@ -0,0 +1,37 @@
+defmodule AboutProcesses do
+  use Koans
+
+  think "You are a process with a process ID" do
+    pid_name = inspect(self)
+
+    assert_? pid_name =~ ~r/#PID<\d+\.\d+\.\d+>/
+  end
+
+  think "Processes receive messages" do
+    send self, {:hello, "world"}
+
+    receive do
+      {:hello, message} -> assert message == __?
+    end
+  end
+
+  think "A spawned process is a function running in parallel" do
+    spawn __?
```

Comment:
Ya, I tried to go the capture route as well. Was just confusing. I think I like "you spawn functions, write a function that is interesting to you"